network/controller/bootstrap/bootstrap.go

```diff
@@ -221,7 +222,11 @@ func (bc *bootstrapper) Bootstrap(ctx context.Context) (*network.BootstrapResult
 	log.Info("Bootstrapping to discovery node")
 	ctx, span := instracer.StartSpan(ctx, "Bootstrapper.Bootstrap")
 	defer span.End()
-	ch := bc.getDiscoveryNodesChannel(ctx, bc.Certificate.GetDiscoveryNodes(), 1)
+	discoveryNodes := bc.Certificate.GetDiscoveryNodes()
+	if utils.OriginIsDiscovery(bc.Certificate) {
+		discoveryNodes = RemoveOrigin(discoveryNodes, bc.NodeKeeper.GetOrigin().ID())
```

Comment:
What we should do when `discoveryNodes` becomes empty?

Reply:
we should fail the bootstrap process, fixed